hack/test-stirunimage.sh

```diff
@@ -82,11 +82,6 @@ test_debug "s2i build with relative path without file://"
 s2i build cakephp-ex openshift/php-55-centos7 test --loglevel=5 &> "${WORK_DIR}/s2i-rel-noproto.log"
 check_result $? "${WORK_DIR}/s2i-rel-noproto.log"
 
-test_debug "s2i build with relative path with file://"
-
-s2i build file://./cakephp-ex openshift/php-55-centos7 test --loglevel=5 &> "${WORK_DIR}/s2i-rel-proto.log"
-check_result $? "${WORK_DIR}/s2i-rel-proto.log"
```

Comment:
yes, under the section marked "s2i build with relative path without file://" :-)